src/Tests/Topology/When_configuring_publishers.cs

```diff
@@ -34,7 +34,7 @@ public void Should_not_map_publisher_and_type_if_it_is_not_a_message()
 
             var exception = Assert.Throws<InvalidOperationException>(() => configuration.GetPublishersFor(typeof(MyType)));
             StringAssert.Contains($"No publishers configured for `{typeof(MyType).FullName}`", exception.Message);
-            StringAssert.Contains($"Please add mappings via `endpointConfiguration.UseTransport<AzureServiceBusTransport>().UseTopology<EndpointOrientedTopology>().RegisterPublisherForType(\"endpointname\", typeof({typeof(MyType).FullName}))`", exception.Message);
+            StringAssert.Contains($"Register mappings via `endpointConfiguration.UseTransport<AzureServiceBusTransport>().UseTopology<EndpointOrientedTopology>().RegisterPublisherForType(\"endpointname\", typeof({typeof(MyType).FullName}))`", exception.Message);
```

Comment:
Ok, you're right. I already test exception message with the test above. I can remove the assert about message from this test.

>The name of the test implies that nothing should happen if registered type is not a message.
Yes, but the only way to verify that a "not message" hasn't been registered is to try to "resolve" it, asserting against an exception. Others ideas?